modules/bubblewrap.nix

```diff
@@ -78,20 +84,34 @@ in {
       type = with types; attrsOf (nullOr str);
       default = {};
     };
+
+    monitor = mkOption {
```

Comment:
I guess since the description says flatpak-session-helper, this option should probably be in flatpak-shim, as `options.flatpak.session-helper.enable`, even if we're only using its file monitor functionality for now. Enabling it by default based on networking being enabled is a tough sell, since this goes as far as running an extra service on the host.

Reply:
I'll move it to the shim module. I set it to default to network because it replaces the networking bind mounts. I can remove it, but it will require everyone to change their configuration to keep network access working.

Reply:
Ideally we'd support both variants, otherwise it might seem like network access *requires* flatpak-session-helper.

Reply:
Ok I see, I had removed the old networking bind mounts expecting this to replace them, but I'll add them back so both options work.